colors.css

```diff
@@ -39,13 +40,13 @@
   --border-grey: #e5e7eb;
   --border-input: #d0d0d0;
   --text-input: #afafaf;
-  --dark-blue: rgba(17,24,39,1);
+  --dark-blue: rgba(17, 24, 39, 1);
   --box-shadow: rgba(0, 0, 0, 0.3);
```

Comment:
use either `hex` or `rgba` color codes and keep them constant throughout

Reply:
yaah, makes sense. Pushing this change.

Reply:
awesome